src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +19,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl();
```

Comment:
The `CountryServiceImpl` should be initialized with the `sessionFactory` to ensure it can manage Hibernate sessions correctly. Modify the constructor to accept a `SessionFactory` parameter.